acceptance-tests/pom.xml

```diff
@@ -17,7 +17,7 @@
         <jenkins.acceptance-test-harness.version>1.89</jenkins.acceptance-test-harness.version>
         <bitbucket.version>7.8.0</bitbucket.version>
         <httpclient.version>4.5.10</httpclient.version>
-        <jenkins.version>2.204.6</jenkins.version>
+        <jenkins.version>2.222.4</jenkins.version>
```

Comment:
I did some poking around the plugin versions and they've all upgraded to 2.222.1- either Jenkins isn't using semver or there's some other reason they're not going higher. I don't think it makes a huge difference, but we might want to go 2.222.1 just for consistency

Reply:
I think most plugins, like us, try the lowest possible version. I think Jenkins is _mostly_ semver but not always so it is safer to play it low on versions. I say if this works, let's stay on `2.222.4`